src/sortStudents.ts

```diff
@@ -1,16 +1,66 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function getAvgGrade(grades: number[]): number {
+  const sumOfGrades: number = grades
+    .reduce((sum: number, current: number) => sum + current);
```

Comment:
Not necessary for tests, but it would be nice to set initial zero value for reducer
```suggestion
    .reduce((sum: number, current: number) => sum + current, 0);
```